build/cmake/options.cmake

```diff
@@ -114,6 +114,7 @@ wx_add_thirdparty_library(wxUSE_EXPAT EXPAT "use expat for XML parsing" DEFAULT_
 wx_add_thirdparty_library(wxUSE_LIBJPEG JPEG "use libjpeg (JPEG file format)")
 wx_add_thirdparty_library(wxUSE_LIBPNG PNG "use libpng (PNG image format)")
 wx_add_thirdparty_library(wxUSE_LIBTIFF TIFF "use libtiff (TIFF file format)")
+wx_add_thirdparty_library(wxUSE_NANOSVG NanoSVG "use nanosvg for svg conversion")
```

Comment:
Ideally these new options are added to `setup.h(.in)`, like `wxUSE_LIBTIFF` so it is availble for all build systems. And then CMake can enable it when using external nanosvg.